templates/jetty/files/.platform/services.yaml

```diff
@@ -0,0 +1,8 @@
+# The services of the project.
+#
+# Each service listed will be deployed
+# to power your Platform.sh project.
+
+#database:
```

Comment:
Per discussion elsewhere let's standardize this naming.

Why is it not enabled here by default?